src/main/java/com/softserve/academy/spaced/repetition/DTO/impl/CardRatingPublicDTO.java

```diff
@@ -15,7 +15,7 @@ public String getAccountEmail() {
     }
 
     public long getCardId() {
-        return getEntity().getCardId();
+        return getEntity().getCard().getId();
```

Comment:
Why was this change required?

Reply:
Because I added refence key to table CardRating and to get CardId we need call getCard() method.